src/migrations/20230221212034_CreateTimestamp.ts

```diff
@@ -0,0 +1,22 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  const tableNames = [
+    'account_wallet_mappings',
+    'attestations_completed',
+    'escrow',
+    'transfers',
+  ]
+  await Promise.all(
+    tableNames.map(
+      async (tableName) =>
+        await knex.schema.alterTable(tableName, (table) =>
+          table.bigInteger('timestamp'),
+        ),
+    ),
+  )
+}
+
+export async function down(knex: Knex): Promise<void> {
+  throw new Error('Unable to revert migration')
```

Comment:
I see that we do this (throw an error) for a handful of migrations currently, but most are able to cleanly revert; is there a reason why this one in particular should throw an error?

Reply:
seems like it would be pretty easy to just drop the new column

Reply:
No strong reason other than to match the approach of previous migrations. Updated the code to drop the column